docs/en/observability/apm/data-model.asciidoc

```diff
@@ -50,6 +50,8 @@ When this occurs, the {apm-app} will display the number of spans dropped.
 
 To configure the number of spans recorded per transaction, see the relevant Agent documentation:
 
+// Is this still unsupported in Android and iOS?
+* Android: _Not yet supported_
```

Comment:
Should Android/iOS be listed as _Not yet supported_ in the list in [Dropped spans](https://observability-docs_3507.docs-preview.app.elstc.co/guide/en/observability/master/data-model-spans.html#data-model-dropped-spans)?

Reply:
still not supported